### ISSUE TITLE
OrbitalImages (estimator for orbital visualization) is undocumented 

### ISSUE THREAD

**Issue by prckent** [QMCPACK/qmcpack#778]
The name should also be revisited and a test added. e.g. OrbitalVisualizer. "OrbitalImages" is not obviously a postprocessor and not easily found.

**Comment by jtkrogel** [comment]
Agreed.  The move away from "Estimator" classification for this operation may require some agreement/framework for introducing pre/post-processors to the code.  This one needs all of the init but none of the execution.

**Comment by ye-luo** [comment]
I have a SPOSetScanner.h implemented as a tool for orbitals. It has less functionality than OrbitalImages but can be used right after the construction of SPO. Consider merging these two.

**Comment by prckent** [comment]
Still relevant. It exists only as an entry in a input option table. https://qmcpack.readthedocs.io/en/develop/hamiltonianobservable.html#general-estimators . The code is very readable src/QMCHamiltonians/OrbitalImages.*
